www/devtools.inc.php

```diff
@@ -831,19 +833,43 @@ function ParseDevToolsEvents(&$json, &$events, $filter, $removeParams, &$startOf
                 $previousStepIndex = $currentStepIndex;
                 $currentStepIndex++;
                 $thisStepRequestId = null;
+                $thisStepPushBackRequestId = null;
+                $thisStepResponseReceived = false;
```

Comment:
No, `$thisStepPushBackRequestId` holds the first request id value whereas, `$thisStepResponseReceived` is just a flag to indicate the response for the first request has been received.